example/README.md

```diff
@@ -1,17 +1,118 @@
 # HPA Example
 
-Deploy the example resources in [./resources.yaml] to a Kubernetes cluster:
+## Disclaimer
+This example is validated using the latest SignalFX HPA Adapter on kubernetes v1.20. <br/>
+The adapter implements the autoscaling api `v2beta1`, this version is not supported in kubernetes v1.22 and above. <br/>
```

Comment:
Based on this, while the adapter was validated on v1.20, since the autoscaling api is supported on v1.21 - should we expect any issues with it incase the customer is using a mix of k8s versions?

Reply:
based on this https://github.com/kubernetes-sigs/metrics-server#compatibility-matrix , It "should" work, but I have not tested it